src/Native/build-native.sh

```diff
@@ -2,11 +2,13 @@
 
 usage()
 {
-    echo "Our parameters changed! The parameters: buildArch, buildType and HostOS"
-    echo "are passes by the Run Command Tool."
+    echo "Our parameters changed! The parameters: buildArch, buildType, buildOS, numProc"
+    echo "are passed by the Run Command Tool."
+    echo "If you plan to only run this script, be sure to pass those parameters in the stated order."
```

Comment:
I see that numProc needs to be passed in as the 4th parameter. Can we just parse it from a `--num-proc` arg? TBH I'm not sure what I'm even supposed to pass for this.